include/fastdds/dds/xtypes/type_representation/detail/dds-xtypes_typeobject.idl

```diff
@@ -247,7 +247,7 @@ module DDS { module XTypes {
 
 
     // eProsima extension to support TypeIdentifier no value cases.
-    @final
+    @final @nested
```

Comment:
Code needs to be regenerated after this change.

Reply:
`@nested` is not supported currently by `fastddsgen`. The generated code is the same after this change.